.travis.yml

```diff
@@ -1,13 +0,0 @@
-language: node_js
-node_js:
-    - "10.12.0"
-    - 10
-    - 12
-    - 13
-
-script:
-    - node Makefile.js lint
-    - node Makefile.js test
-
-after_success:
-    - npm run coveralls
```

Comment:
Are we giving up on coveralls?

Reply:
I'm definitely not against it using coveralls. I removed this because this `npm` script doesn't currently exist and we don't have [node-coveralls](https://github.com/nickmerwin/node-coveralls) in our dev dependencies. But maybe I'm missing something? I looked through the Travis docs to make sure there wasn't some magic happening here, and I couldn't find anything.

Reply:
Ah ok. Just wanted to make sure we weren't missing anything.